### PR TITLE
Task dl/j310 adjust login screen in accordance with the mockup

### PR DESCRIPTION
**Pull request by Denidzio**
image

### CODE REVIEW

File: src/components/PlainText/styles.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ type Styles = {
 
 const styles = StyleSheet.create<Styles>({
   text: {
-    fontFamily: fonts.MainMedium
+    fontFamily: fonts.MainRegular
```
Comment (EugeniusGR) [inline src/components/PlainText/styles.ts:10]: we have PlainText for this, dont we?

**Review by EugeniusGR — CHANGES_REQUESTED** [review]
other lgtm

File: src/components/PlainText/styles.ts (hunk 1 above)
Comment (Denidzio) [reply src/components/PlainText/styles.ts:10]: How I noticed, main `fontWeight` of out layout is `MainRegular`.

Comment (EugeniusGR) [reply src/components/PlainText/styles.ts:10]: mb we will use it in the future, mb it will be good idea to create type for PlainText

**Review by eugeneprib — APPROVED** [review]

**Pull request merged**